src/providers/ldap/ldap_child.c

```diff
@@ -61,7 +61,6 @@ static void sig_term_handler(int sig)
 static krb5_context krb5_error_ctx;
 #define LDAP_CHILD_DEBUG(level, error) KRB5_DEBUG(level, krb5_error_ctx, error)
 
-static const char *__ldap_child_krb5_error_msg;
```

Comment:
@jhrozek Thanks for Comment.
My new changes are:
1. Removing "static const char *__ldap_child_krb5_error_msg;"
2. Removing #define KRB5_SYSLOG(krb5_error) definition
Which I have already done, as I can see.

What changes are you referring to?